#! /bin/sh /usr/share/dpatch/dpatch-run
## 81-dont-test-for-long-names.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: remove AC_SYS_LONG_FILE_NAMES test to avoid problems with fakeroot
## DP: this is a _very_ old patch from Cyrus 2.1, might not be needed anymore

@DPATCH@
--- cyrus-imapd-2.4.orig/configure.in
+++ cyrus-imapd-2.4/configure.in
@@ -99,10 +99,12 @@ AC_AIX
 AC_ISC_POSIX
 AC_PROG_AWK
 AC_C_CONST
-AC_SYS_LONG_FILE_NAMES
-if test $ac_cv_sys_long_file_names = no; then
-	AC_MSG_ERROR(The Cyrus IMAPD requires support for long file names)
-fi
+dnl fakeroot sometimes fails this test
+dnl and Debian always supports long filenames anyway
+dnl AC_SYS_LONG_FILE_NAMES
+dnl if test $ac_cv_sys_long_file_names = no; then
+dnl	AC_MSG_ERROR(The Cyrus IMAPD requires support for long file names)
+dnl fi
 AC_C_INLINE
 
 dnl Check the size of various types
